packages/openactive-integration-tests/test/helpers/request-state-helper.js

```diff
@@ -0,0 +1,118 @@
+const RequestHelper = require("./request-helper");
+const pMemoize = require("p-memoize");
+
+class RequestStateHelper {
+  constructor (logger) {
+    this.requestHelper = new RequestHelper(logger);
+    this.logger = logger;
+  }
+
+  log (msg) {
+    if (!this.logger) return;
+
+    this.logger.log(msg);
+  }
+
+  // var eventId;
+  // var opportunityId;
+  // var offerId;
+  // var sellerId;
+  // var uuid;
+  // var totalPaymentDue;
+  // var orderItemId;
+  //
+  // var c1Response;
+  // var c2Response;
+  // var bResponse;
+  // var uResponse;
+  // var getOrderPromise;
+  // var ordersFeedUpdate;
```

Comment:
I think it may be worth keeping a list of the state set but not as a commented out list of variable declarations